examples/lb3-application/src/__tests__/acceptance/lb3app.acceptance.ts

```diff
@@ -138,7 +138,7 @@ describe('CoffeeShopApplication', () => {
     let apiSpec: OpenApiSpec;
 
     before(async () => {
-      apiSpec = app.lbApp.restServer.getApiSpec();
+      apiSpec = await app.lbApp.restServer.getApiSpec();
```

Comment:
This is a breaking change. Please update the commit message per https://loopback.io/doc/en/lb4/code-contrib-lb4.html#footer-optional. See an example at https://github.com/strongloop/loopback-next/pull/4451/commits/98a3f422568877a3b813160186eec9630208319f and https://github.com/strongloop/loopback-next/blob/master/packages/context/CHANGELOG.md#breaking-changes.

Reply:
Please check the commit message is acceptable.